packages/trader/src/AppV2/Components/DraggableList/draggable-list.tsx

```diff
@@ -83,10 +107,15 @@ const DraggableList: React.FC<DraggableListProps> = ({ categories, onRightIconCl
                                                 ref={provided.innerRef}
                                                 {...provided.draggableProps}
                                                 {...provided.dragHandleProps}
+                                                className={`draggable-list-category__item ${
+                                                    draggedItemId === item.id ? 'is-clicked' : ''
```

Comment:
Just in case if you want: we add `clsx` to trader package.json, so you can do smth like this:
`className={clsx('draggable-list-category__item', draggedItemId === item.id && 'draggable-list-category__item--clicked')}`
Also, people love when modifiers from BEM are written with '--'
It's just more about taste

Reply:
Hmm... Sure, I actually think this class is not used anymore, so I will remove it